lib/trino-filesystem-alluxio/src/main/java/io/trino/filesystem/alluxio/AlluxioCacheStats.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.filesystem.alluxio;
+
+import com.google.errorprone.annotations.ThreadSafe;
+import io.airlift.stats.DistributionStat;
+import io.trino.filesystem.Location;
+import org.weakref.jmx.Managed;
+
+@ThreadSafe
+public class AlluxioCacheStats
+{
+    private final DistributionStat externalReads = new DistributionStat();
+    private final DistributionStat cacheReads = new DistributionStat();
+
+    @Managed
+    public DistributionStat getExternalReads()
+    {
+        return externalReads;
+    }
+
+    @Managed
+    public DistributionStat getCacheReads()
+    {
+        return cacheReads;
+    }
+
+    public void recordCacheRead(Location location, int length)
+    {
+        cacheReads.add(length);
+    }
+
+    public void recordExternalRead(Location location, int length)
```

Comment:
why do you need a `Location location` if you don't use it? Drop it

Reply:
It's used for testing, but I'll see if I can find a cleaner way.